freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/FreeIpaV1Endpoint.java

```diff
@@ -302,4 +305,14 @@ FreeIpaRecommendationResponse getRecommendation(
             @ValidCrn(resource = CrnResourceDescriptor.CREDENTIAL) @QueryParam("credentialCrn") @NotEmpty String credentialCrn,
             @QueryParam("region") @NotEmpty String region,
             @QueryParam("availabilityZone") String availabilityZone);
+
+    @PUT
+    @Path("vertical_scaling")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = VERTICAL_SCALE_BY_CRN, produces = MediaType.APPLICATION_JSON, notes = FreeIpaNotes.FREEIPA_NOTES,
+            nickname = "putVerticalScalingFreeIPAV1ByEnvironmentCrn")
+    VerticalScaleResponse putVerticalScalingFreeIPAV1ByEnvironmentCrn(
```

Comment:
could you rename `FreeIPA` to `FreeIpa` here too? thanks. both the nickname and the method